api/src/resources/bucket.rs

```diff
@@ -70,11 +71,12 @@ impl From<Id> for Identifier {
 
 impl FromStr for Identifier {
     type Err = Error;
-
     fn from_str(string: &str) -> Result<Self> {
+        let full_name_regex = Regex::new("^[^/]+/[^/]+$").unwrap();
```

Comment:
Do we not want to at least attempt to mimic the constraints in the store for this regex? Currently the bucket name regex in the backend is `^[A-Za-z0-9-_]{1,256}/[A-Za-z0-9-_]{1,256}$`. We could simplify it and skip the size and just do `+`, but I think we should at least constraints the character set, no?

Reply:
PS. I know we don't need to be very precise here since the backend will catch the errors anyway, but since we're here already I don't see why not? I doubt we'll ever relax the backend regex more than it is so we might as well fail in the CLI.